src/lib/data/createables.ts

```diff
@@ -1615,6 +1615,499 @@ const Createables: Createable[] = [
 			[itemID('Castle wars ticket')]: 75
 		}
 	},
+	/** Revert Graceful */
```

Comment:
leaving again as a review so github knows changes are requested.

I think these should be 'generated' rather than have every single graceful piece written out like this

something like this:
```js
for (const piece of gracefulTops.map(getOSItem)) {
    createables.push({{name: `Revert ${piece.name}`, input..., output...});
}
```